functions-and-scope.js

```diff
@@ -16,6 +16,18 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+//1. Check of cijfer >= 8
+
+const gradesCumLaude = grades.slice(); //creëer een referentieloze lijst op basis van de grades lijst
+
+for (let i = 0; i < gradesCumLaude.length; i++) {
+    if (gradesCumLaude[i] < 8) { //als cijfer in lijst is lager dan 8
+        gradesCumLaude.splice(i, 1); //dan verwijder i (1 item) en voeg 0 items toe.
+        i--; //verlaag i met 1, omdat de lijst met 1 is afgenomen
+    }
+}
+
```

Comment:
Het is niet goed om in een for-loop nog eens de array aan te passen. Ik had liever gezien dat je een counter had opgehoogd zodra het cijfer een 8 of hoger zou zijn. 

```
let counter = 0;
for (let i = 0; i < gradesCumLaude.length; i++) {
    if (gradesCumLaude[i] >= 8){
        counter++;
    }
}
```